test/modules/extensions/terrain/terrain-cover.spec.ts

```diff
@@ -147,7 +147,7 @@ test('TerrainCover#viewport diffing#non-geo', async t => {
   t.end();
 });
 
-test('TerrainCover#layers diffing#non-geo', async t => {
+test.skip('TerrainCover#layers diffing#non-geo', async t => {
```

Comment:
Just double-checking, are these skipped intentionally?

Reply:
Yes, the current harness needs some updates to get default effect registration working. I will make the change in a follow-up PR.